_layouts/default.html

```diff
@@ -15,6 +15,10 @@
     <link rel="stylesheet" href="{{ '/assets/css/ie.css' | relative_url }}">
     <![endif]-->
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
+    <!-- Setup theme-color. -->
+    <meta name="theme-color" content="#353535">
+    <meta name="msapplication-navbutton-color" content="#353535">
+    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
```

Comment:
Thank you for this! I am not sure what kind of approach is best. Since we allow folks to override the colors in the SCSS files, should we find a way to allow users to modify this, too? Maybe make it an include, or use variables in a data file?

Reply:
I'm also not sure what is better — a custom include has the advantage that users may want to tune more concerning colours than can be exposed by variables, for example there is no direct mapping from the colour you may want to use for the header and the `apple-style` colour such as `black-translucent`. Of course, a data file has the advantage that all settings could be kept in one place. 

I'd currently tend towards an include, but I don't have a strong opinion on this. Given this reasoning, do you prefer one over the other? Then I could surely rework this :wink:.

Reply:
Let's use an include! Let's call it something like `theme_color_meta_headers.html` or something?

Reply:
Done, rebased to master and force-pushed, let me know if you prefer the commits to be squashed :smile:.